src/Servers/Kestrel/Core/src/Internal/Http3ConnectionContext.cs

```diff
@@ -1,44 +1,25 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Buffers;
 using System.Net;
 using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http.Features;
-using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3;
-using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal
 {
-    internal class Http3ConnectionContext
+    internal class Http3ConnectionContext : BaseHttpConnectionContext
     {
         public Http3ConnectionContext(
             string connectionId,
+            HttpProtocols protocols,
```

Comment:
Nit: It looks like we could remove param this and just pass `HttpProtocols.Http3` directly to `base`.